Source/Meadow.Foundation.Peripherals/Sensors.Distance.A02yyuw/Samples/A02yyuw_Sample/MeadowApp.cs

```diff
@@ -8,7 +10,12 @@
 namespace A02yyuw_Sample
 {
     // Change F7FeatherV2 to F7FeatherV1 for V1.x boards
+#if DEVICE_FEATHER          // Feather: Change F7FeatherV2 to F7FeatherV1 for V1.x boards
```

Comment:
This is an interesting idea, we should think about something like this and apply it to all samples in the repo.

Reply:
We have a cleaner pattern for this that doesn't require compiler directives.  See this as an example:
https://github.com/WildernessLabs/Meadow.Foundation/blob/fb357c4daf2124adeab0a450e663dc3f691c2ba9/Source/Meadow.Foundation.Peripherals/Sensors.Atmospheric.Ahtx0/Samples/Aht10_Sample/MeadowApp.cs#L10